### ISSUE TITLE
Build failure due to deprecated Pandoc -S switch

### ISSUE THREAD

**Issue by ntwk** [jagregory/abrash-black-book#36]
Attempting to build the HTML or EPUB versions of the book fails with the following error:

```
--smart/-S has been removed.  Use +smart or -smart extension instead.
For example: pandoc -f markdown+smart -t markdown-smart.
Try pandoc --help for more information.
```

I am currently using Pandoc 2.2.3.2 and it appears that Pandoc 2.0 made some changes to its command-line interface.  The same issue was seen in  [rmarkdown issue #1120](https://github.com/rstudio/rmarkdown/issues/1120).

**Comment by gerroon** [comment]
Same here

 make epub
mkdir -p out
rm -f out/black-book.epub
pandoc -S --to epub3 -o out/black-book.epub --epub-cover-image images/cover.png --toc --epub-chapter-level=2 --data-dir=epub --template=epub/template.html src/intro.md src/about.md src/about_author.md src/chapter-01.md src/chapter-02.md src/chapter-03.md src/chapter-04.md src/chapter-05.md src/chapter-06.md src/chapter-07.md src/chapter-08.md src/chapter-09.md src/chapter-10.md src/chapter-11.md src/chapter-12.md src/chapter-13.md src/chapter-14.md src/chapter-15.md src/chapter-16.md src/chapter-17.md src/chapter-18.md src/chapter-19.md src/chapter-20.md src/chapter-21.md src/chapter-22.md src/chapter-23.md src/chapter-24.md src/chapter-25.md src/chapter-26.md src/chapter-27.md src/chapter-28.md src/chapter-29.md src/chapter-30.md src/chapter-31.md src/chapter-32.md src/chapter-33.md src/chapter-34.md src/chapter-35.md src/chapter-36.md src/chapter-37.md src/chapter-38.md src/chapter-39.md src/chapter-40.md src/chapter-41.md src/chapter-42.md src/chapter-43.md src/chapter-44.md src/chapter-45.md src/chapter-46.md src/chapter-47.md src/chapter-48.md src/chapter-49.md src/chapter-50.md src/chapter-51.md src/chapter-52.md src/chapter-53.md src/chapter-54.md src/chapter-55.md src/chapter-56.md src/chapter-57.md src/chapter-58.md src/chapter-59.md src/chapter-60.md src/chapter-61.md src/chapter-62.md src/chapter-63.md src/chapter-64.md src/chapter-65.md src/chapter-66.md src/chapter-67.md src/chapter-68.md src/chapter-69.md src/chapter-70.md src/appendix-a.md src/copyright.md
--smart/-S has been removed.  Use +smart or -smart extension instead.
For example: pandoc -f markdown+smart -t markdown-smart.
Try pandoc --help for more information.
make: *** [Makefile:15: epub] Error 2

**Comment by rockerdaddy** [comment]
Solution until the project fixes it:

Edit the file Makefile. Lines 10 and 15 start like this:

`10    pandoc -S --to=html5`

 Change them to:

`10    pandoc --to=html5`

 The -S is for smart quotes, em- and en-dashes, and ellipses. The smart extension is not supported for html5 or epub. Personally, I can live without the fancy typography.

**Comment by jagregory** [comment]
That looks sensible. If you send a PR with that change I’ll merge it, otherwise I’ll take a look when I have some time soon.

**Comment by jagregory** [comment]
Fixed in #37
